CalibFormats/SiPixelObjects/src/PixelFEDCard.cc

```diff
@@ -253,7 +253,6 @@ PixelFEDCard::PixelFEDCard(vector<vector<string> > &tableMat) : PixelConfigBase(
 
   // Read below quantities pertaining to a single FED that are equal accross 36 channels
   if (first) {
```

Comment:
Usage of the variable `first` could be completely removed: @cms-sw/trk-dpg-l2 please suggest if either remove all appearances of `first` in this code, or fix it so that `first` gets the meaning that was originally intended for it

Reply:
@cms-sw/trk-dpg-l2 gentle ping

Reply:
It looks like this piece of code was part of some loop, but in its current form the variable `first` is completely redundant and I agree with @perrotta that it can be removed. As far as I can tell, this was introduced in the code back in 2008 (https://github.com/cms-cvs-history/CalibFormats-SiPixelObjects/commit/e136e402f88fe8b34a2c6511353cb2a95ed2930d) and has been there ever since, and the code has not been in active development for the last 10 years (https://github.com/cms-sw/cmssw/commits/master/CalibFormats/SiPixelObjects/src/PixelFEDCard.cc). Just remove the variable and make the static analyzer happy :)